scripts/tests/chiptest/yamltest_with_chip_repl_tester.py

```diff
@@ -15,22 +15,23 @@
 #    limitations under the License.
 
 import atexit
+import builtins
 import logging
 import os
 import tempfile
 import traceback
 
 # isort: off
 
-from chip import ChipDeviceCtrl  # Needed before chip.FabricAdmin
+from chip import ChipDeviceCtrl  # noqa: F401 # Needed before chip.FabricAdmin
```

Comment:
What does that noqa bit do?

Reply:
ChipDeviceCtrl needed before chip.FabricAdmin but is not used directly in this file. Flake8 reports the problem and noqa allows you to bypass this line.

Reply:
I think that info should probably go into the comment (that F401 is "Module imported but unused", etc).